layouts/index.html

```diff
@@ -119,7 +119,7 @@ <h2 class="font-medium">Getting started</h2>
               </p>
             </div>
             <div>
-              <a href="/guides/getting-started/"
+              <a href="/get-started/introduction/"
```

Comment:
Linking to the guide seems right. Just noting that if the users click "Get started" here, they're taken to the introduction. But if they click "Get started" on the top nav, they're taken to the overview. Both are "Get started", but go to different places.